tests/test_connection.py

```diff
@@ -753,6 +753,20 @@ def test_funding_toolarge(node_factory, bitcoind):
     l1.rpc.fundchannel(l2.info['id'], amount)
 
 
+def test_funding_by_utxos(node_factory, bitcoind):
+    """Fund a channel with specific utxos"""
+    l1, l2 = node_factory.line_graph(2, fundchannel=False)
+
+    # Get 2 differents utxo
+    l1.fundwallet(0.01 * 10**8)
+    l1.fundwallet(0.01 * 10**8)
+    wait_for(lambda: len(l1.rpc.listfunds()['outputs']) == 2)
+
+    # Then fund a channel with them
+    utxos = [utxo["txid"] + ":" + str(utxo["output"]) for utxo in l1.rpc.listfunds()["outputs"]]
+    l1.rpc.fundchannel(l2.info["id"], utxos)
```

Comment:
OK, test that you can't grab some bogus UTXO too?  And maybe an already-consumed UTXO?

Reply:
Ok, how could I get a bogus UTXO ?

Reply:
Dump 32 bytes from `/dev/random` for txid plus 2 bytes for outnum?

Reply:
I didn't see your comment and finally added a test that makes a node funding a channel with utxos it doesn't own, is it OK ? It takes the same code path.

Reply:
It is fine.